tools/codegen/config.yml

```diff
@@ -0,0 +1,45 @@
+resources:
+  push_based_log_export:
+    read:
+      path: /api/atlas/v2/groups/{groupId}/pushBasedLogExport
+      method: GET
+    create:
+      path: /api/atlas/v2/groups/{groupId}/pushBasedLogExport
+      method: POST
+    schema:
+      aliases:
+        group_id: project_id
+      ignores: ["links"]
+      timeouts: ["create", "update", "delete"]
+      
+      # overrides:
```

Comment:
When do you plan to add support for overriding plan modifiers, validators and computability?
Is the plan to do that in master directly?